server/src/controllers/profile.controllers/getProfile.ts

```diff
@@ -35,11 +32,27 @@ export const getProfile = async (req: Request, res: Response, next: NextFunction
             username,
             photo: photoPath,
             email,
-            postsAll,
         };
 
         res.json({userProfile});
     } catch (e) {
         next(e);
     }
 };
+
+export const getProfilePosts = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
+    try {
+        const { userName, page }: IParams = req.params;
+        const user = await getUserByUsername(userName, next);
+
+        const {posts}: any = user;
+        const POSTS_ON_PAGE = 9;
+
```

Comment:
Why don't we set this param dynamically from ui as query param  for example

Reply:
This parameter is constant for all project. It divides all posts into small parts for better performance.